public/app/features/alerting/unified/components/rule-editor/GrafanaEvaluationBehavior.tsx

```diff
@@ -271,6 +277,35 @@ export function GrafanaEvaluationBehavior({
           evaluateEvery={evaluateEvery}
         />
         <ForInput evaluateEvery={evaluateEvery} />
+
+        {existing && (
```

Comment:
This was proposed by Deyan, the reason is that this is a feature that will be used mostly by people coming from legacy alerting to prevent previously existing paused alerts to be evaluated and create noise when they shouldn't. For new alerts this shouldn't be necessary and we don't want to over-advertise this feature too much as it overlaps with Silences.

Reply:
Ok. So, in this case, maybe we should add an info tooltip in which we recommend switching to Silences or Mute timings instead of pausing alerts?

Reply:
I think we can do that in a later iteration of the design 👍